runtime/doc/map.txt

```diff
@@ -272,6 +274,29 @@ again for using <expr>.  This does work: >
 Using 0x80 as a single byte before other text does not work, it will be seen
 as a special key.
 
+						*<Cmd>* *:map-command*
+A command mapping is a mapping that directly executes a command. Command
+mappings are written by placing a command in between `<Cmd>` and `<Cr>` in the
+rhs of a mapping (in any mode):
+  noremap <f3> <Cmd>echo mode(1)<cr>
+
+							*E5520*
+The command must be complete and ended with a `<Cr>`. If the command is
```

Comment:
Don't need backticks around keycodes in the help. Also it's spelled `<CR>` everywhere, `:helpgrep <Cr>` returns no results. 

Should also use `<CR>` in the error messages.

Reply:
Done so!